_about/employment-opportunities.html

```diff
@@ -0,0 +1,29 @@
+---
+title: Employment Opportunities
+---
+
+<p class="mt-3">
+    The OSG Consortium is proud to display the following positions dedicated to advancing the state of research computing.
+</p>
+
```

Comment:
I would add an H2 here, something along the lines of “OSG Project Positions”, with a intro line of something like “The following positions are with projects that contribute directly to the OSG program of work.”